pkg/skaffold/deploy/kubectl_test.go

```diff
@@ -491,12 +493,26 @@ func TestKubectlRender(t *testing.T) {
 					Tag:       "gcr.io/k8s-skaffold/skaffold:test",
 				},
 			},
+			labels: []Labeller{},
 			input: `apiVersion: v1
 kind: Pod
+metadata:
```

Comment:
was there a reason for adding this in input?
```
  metadata
     namespace: default
```

Reply:
Yes! If `metadata` doesn't exist, the default labeller doesn't execute, therefore not adding any labels.

Without it `skaffold.dev/deployer: kubectl` won't be added.